MAUI/TimePicker/customizations.md

```diff
@@ -0,0 +1,403 @@
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timepicker.ColumnHeaderView = new TimePickerColumnHeaderView()
+{
+    DividerColor = Color.FromArgb("#A9A9A9")
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customization of column header
+
+You can customize the column header view text style and background color of the `Time Picker` using the [TextStyle] and [Background] properties of the [TimePickerColumnHeaderView].
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.ColumnHeaderView >
+        <picker:TimePickerColumnHeaderView Background="#6750A4">
+            <picker:TimePickerColumnHeaderView.TextStyle >
+                <picker:PickerTextStyle FontSize="15" TextColor="White" />
+            </picker:TimePickerColumnHeaderView.TextStyle>
+        </picker:TimePickerColumnHeaderView>
+    </picker:SfTimePicker.ColumnHeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.ColumnHeaderView = new TimePickerColumnHeaderView()
+{
+    Background = Color.FromArgb("#6750A4"),
+    TextStyle = new PickerTextStyle()
+    {
+        TextColor = Colors.White,
+        FontSize = 15,
+    }
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Footer customization
+
+You can customize the time picker footer view by using the `FooterView` property of `SfTimePicker`.
+
+### Set footer with ok and cancel button customizations
+
+In the SfTimePicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] properties of [PickerFooterView]. It allows you to confirm or cancel the selected Time. The `OkButtonText` can be enabled using the [ShowOkButton] property in the [PickerFooterView].
+The Default value of [OkButtonText] property is "OK", [CancelButtonText] is "Cancel". To enable the footer view by setting the [Height] property grater than 0 of [PickerFooterView]. The default value of `Height` property is 0.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.FooterView >
+        <picker:PickerFooterView Height="40" OkButtonText="Ok Button"
+                                 CancelButtonText="Cancel Button"/>
+    </picker:SfTimePicker.FooterView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.FooterView = new PickerFooterView()
+{
+    Height = 40,
+    OkButtonText = "Ok Button",
+    CancelButtonText = "Cancel Button",
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Set divider color
+
+The SfTimePicker control allows you to customize the footer divider color by setting the [DividerColor] property of [PickerFooterView].
```

Comment:
The SfTimePicker control allows you to customize the footer divider color by setting the [DividerColor] property of the [PickerFooterView].